pkg/translate/translate.go

```diff
@@ -14,34 +15,94 @@ type TranslateServiceServer struct {
 	pb.UnimplementedTranslateServiceServer
 }
 
+// ----------------------UploadTranslationFile-------------------------------
+
+type uploadParams struct {
+	Language language.Tag
+	Data     []byte
+	Schema   pb.Schema
+}
+
+func parseUploadParams(req *pb.UploadTranslationFileRequest) (*uploadParams, error) {
+	reqLanguage := req.GetLanguage()
+
+	tag, err := language.Parse(reqLanguage)
+	if err != nil {
+		return nil, fmt.Errorf("parsing language '%s': %w", reqLanguage, err)
```

Comment:
shorter, the verb in simple present form

```suggestion
		return nil, fmt.Errorf("parse language '%s': %w", reqLanguage, err)
```